File: javascript/src/linklocationdata.js
const fs = require('fs')

function loadlinklocations(filepath){    
    let linklocationinput = fs.readFileSync(filepath, 'utf8').trim().split(/\r?\n/)
    let linklocations = []
    for (let i = 0; i < linklocationinput.length ; i += 1 ){
        let casedata = linklocationinput[i].split(' ')
        let x = parseFloat(casedata[0])
        let y = parseFloat(casedata[1])
        let r = parseFloat(casedata[2])
        linklocations.push([x, y, r])
    }
    return linklocations
}

module.exports = {
    loadlinklocations
  }

File: python/src/linklocationdata.py
def loadlinklocations(filepath):   
    linklocationsinput = open(filepath).read().splitlines()
    linklocations = []
    for x in range(0, len(linklocationsinput)):
        casedata = linklocationsinput[x].split()
        x = float(casedata[0])
        y = float(casedata[1])
        r = float(casedata[2])
        linklocations.append([x,y,r])
    return linklocations